doc/introduction/logging.rst

```diff
@@ -0,0 +1,32 @@
+Logging
+=======
+
+PennyLane has support for Python's logging framework. PennyLane defines all logging configuration in a static TOML file, ``log_config.toml``, which can be customized to support rules for a given user or system environment.
+
+To see the default logging options you can explore the contexts of ``log_config.toml`` at the path given from :func:`pennylane.logging.config_path`.
+
+
+Enabling logging
+----------------
+
+To enable logging support with the default options defined in ``log_config.toml`` simply call :func:`pennylane.logging.enable_logging` after importing PennyLane:
+
+.. code:: python
+
+   import pennylane as qml
+   qml.logging.enable_logging()
+   ...
```

Comment:
I think it would be good to have a disclaimer right after this code block that custom loggers should be created _after_ the line `qml.logging.enable_logging()`. This way fewer users will stumble over the same issue that I encountered.

Reply:
Thanks @frederikwilde 
Yep, the plan is to be more explicit in docs in a follow-on PR.